### PR TITLE
Bug fix deploy stage

### PR DESCRIPTION
**Pull request by MaeveOReilly**
-Changed pipeline.yml to remove obsolete services
-Changed pipeline.yml to bind in cloudant which removes need to have credentials in config file
-Changed reference to internal ibm repo to the public one
-Added config-myenv.json so the deploy stage works

### CODE REVIEW

File: config/config-myenv.json [hunk 1]
```diff
@@ -0,0 +1,33 @@
+{
```
Comment (fxulusoy) [inline config/config-myenv.json]: Is this file required? We don't want to push env specific config files into git.

Comment (MaeveOReilly) [reply config/config-myenv.json]: The Deploy stage in Bluemix will fail unless we put one in. But whether we need it or not I'm not sure now - don't need the cloudant stuff but the rest for actions do we?  I don't credentials in there either.

**Comment by fxulusoy** [comment]
@MaeveOReilly This PR is obsolete. Can we close it?

**Comment by MaeveOReilly** [comment]
Yes, I'll close it now.
